.github/workflows/create-action-aqua-security-dependency.yml

```diff
@@ -0,0 +1,74 @@
+name: Create Action aqua-security-dependency
+"on":
+    pull_request:
+        paths:
+            - actions/*
+            - actions/aqua-security-dependency/*
+    push:
+        branches:
+            - main
+        paths:
+            - actions/*
+            - actions/aqua-security-dependency/*
+    release:
+        types:
+            - published
+jobs:
+    create-action:
+        name: Create Action
+        runs-on:
+            - ubuntu-latest
+        steps:
+            - name: Docker login ghcr.io
```

Comment:
Looks like we need to guard against docker login on forks.

Reply:
Ah, needs a rebase.